VK/VK/Animation/StartViewController.swift

```diff
@@ -0,0 +1,24 @@
+// StartViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// StartVC with animation
+final class StartViewController: UIViewController {
+    // MARK: - Life Cycle
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        performSegue(withDuration: 2)
+    }
+
+    // MARK: - Private Methods
+
+    private func performSegue(withDuration: UInt32) {
+        sleep(withDuration)
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let nextViewController = storyboard.instantiateViewController(withIdentifier: "loginViewControllerID")
```

Comment:
```suggestion
        let nextViewController 
```
непонятно что за Некст контролер